lib/components/gene/__tests__/about-test.js

```diff
@@ -0,0 +1,37 @@
+import 'react-native'
+import React from 'react'
+import About from '../about'
+
+// Note: test renderer must be required after react-native.
+import renderer from 'react-test-renderer'
+
+jest.mock('../../opaque_image_view.js', () => 'ImageView')
```

Comment:
so you mock the import basically, and you know that there will be a "box" of that size and positioning there but it's not about what's in that component, right?